code/script.js

```diff
@@ -1,43 +1,214 @@
 // Variables that point to selected DOM elements
 const chat = document.getElementById('chat');
+const inputWrapper = document.getElementById('input-wrapper');
+const nameInput = document.getElementById('name-input');
+const sendBtn = document.getElementById('send')
 
-// If you need any global variables that you can use across different functions, declare them here:
-
-
+//Global variables. I have one for userName to be able to use the value in more than first reply.
+let userName;
 // Declare your functions after this comment
 
 // This function will add a chat bubble in the correct place based on who the sender is
+//without the ${message} the actual message(parameter) in const greetUser will not be passed, only string message. Only the parameter 'bot' 
 const showMessage = (message, sender) => {
-  // the if statement checks if the sender is 'user' and if that's the case it inserts an html senction inside the chat with the posted message
+  // the if statement checks if the sender is 'user' and if that's the case it inserts an html section inside the chat with the posted message
   if (sender === 'user') {
+    console.log("Now the user is replying");
```

Comment:
Great use of console.log()! Do you think you need it still in the code for the end users, if not, you might want to remove it / comment it out? 
(also on line 27)